fix(Classroom Monitor): Make prev/next buttons skip non gradable steps

geoffreykwan opened the pull request:
## Changes
- Previous and next step buttons now skip non gradable steps in the Classroom Monitor

## Test
- Make sure previous and next step buttons skip non gradable steps in the Classroom Monitor
- Make sure previous and next step buttons go to non gradable steps in the Authoring Tool

Closes #475

geoffreykwan committed:
fix(Classroom Monitor): Make prev/next buttons skip non gradable steps

#475

breity reviewed (changes requested):
Great, the arrows skip non-gradable now.

I see one remaining issue: the next arrow is still clickable when teacher is viewing the last gradable step in the unit if there are any non-gradable steps that come after. Clicking on the next arrow then shows "Select a Step" in the StepTools select.

geoffreykwan committed:
fix(Classroom Monitor): Fix previous and next buttons special case

When a non gradable step was at the beginning or end of the project, the
previous and next buttons were clickable even though they shouldn't be.

#475

breity reviewed (approved):
Looks good 👍
